examples/ecdsa_utils/ecdsa_utils.c

```diff
@@ -39,6 +39,10 @@
 // This is a limit of this implementation
 #define DER_INTEGER_MAX_LEN 0x7F
 
+// If highest bit set, we need a different encoding -> 0x7F is maximum
+// This is a limit of this implementation
```

Comment:
The comment is confusing, please try to improve it, e.g., like:
"This implement supports a maximum sequence length of XXX.
The limit comes from ...."